src/openagi/agent.py

```diff
@@ -178,7 +178,8 @@ def worker_task_execution(self, query: str, description: str, task_lists: TaskLi
             cur_task = task_lists.get_next_unprocessed_task()
             worker = self._get_worker_by_id(cur_task.worker_id)
             res, task = worker.execute_task(
-                cur_task,
+                query=query,
```

Comment:
why is query being passed?

Reply:
@shreehari-aiplanet query is passed to fetch relevant information from the knowledge base. I tried passing task.description but it was not giving me appropriate information. Query contains the main intention.